configure.ac

```diff
@@ -95,6 +95,16 @@ else
     fi
 fi
 
+SYSTEM_CUDD=""
+AC_ARG_WITH([cudd],
+	[AS_HELP_STRING([--with-cudd=includedir], [Utilize a system-installed CUDD library])],
```

Comment:
I don't really like this part. `--with-external-cudd` with it being yes or no would be better in my mind. If you need an include directory, then it should be a more specific flag. And then you would expect a matching flag for the library directory.
I would prefer not to take any include directory if possible but test header presence and possibly library presence with AC_CHECK_HEADER and AC_CHECK_LIB. 
This may mean that we have to use macro (like SYSTEM_CUDD) to get the right header location.